Library/Formula/nspr.rb

```diff
@@ -21,10 +21,12 @@ class Nspr < Formula
   def install
     ENV.deparallelize
     cd "nspr" do
-      # Fixes a bug with linking against CoreFoundation, needed to work with SpiderMonkey
-      # See: http://openradar.appspot.com/7209349
-      target_frameworks = (Hardware.is_32_bit? || MacOS.version <= :leopard) ? "-framework Carbon" : ""
-      inreplace "pr/src/Makefile.in", "-framework CoreServices -framework CoreFoundation", target_frameworks
+      if OS.mac?
+        # Fixes a bug with linking against CoreFoundation, needed to work with SpiderMonkey
+        # See: http://openradar.appspot.com/7209349
+        target_frameworks = (Hardware.is_32_bit? || MacOS.version <= :leopard) ? "-framework Carbon" : ""
+        inreplace "pr/src/Makefile.in", "-framework CoreServices -framework CoreFoundation", target_frameworks
+      end
```

Comment:
Is this chunk necessary? It looks as though it shouldn't have any effect on Linux.